lib/internal/modules/esm/module_job.js

```diff
@@ -109,13 +110,14 @@ class ModuleJob {
         if (format === 'commonjs') {
           const importStatement = splitStack[1];
           const namedImports = StringPrototypeMatch(importStatement, /{.*}/)[0];
+          const importsAsPattern = StringPrototypeReplace(namedImports, /\s+as\s+/g, ': ');
```

Comment:
This might look strange, but IIUC it's not possible for `as` (surrounded by whitespace) to appear within an import specifier except when it's actually the `as` keyword, so I didn't think this fix warranted a proper parser.